vendor/assets/javascripts/components/accordion.js

```diff
@@ -157,9 +163,9 @@
         } else {
 
             var tmp = {
-                position   : $ele.css("position"),
-                visibility : $ele.css("visibility"),
-                display    : $ele.css("display")
+                position   : $ele.css('position'),
```

Comment:
This was copied directly from the released 2.27.2 release download. If there's a different location to get these files from that have double-quote formatting, I'd be happy to use that.